hieradata/site/dev/variant/1114S-rev_1.01.yaml

```diff
@@ -0,0 +1,44 @@
+---
+profile::nm::connections:
+  enp1s0f1:
```

Comment:
Please add spec on each host for these interfaces.

Reply:
Updated and fixed tests. PR should be mergeable now.